components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/SessionDataStore.java

```diff
@@ -258,9 +258,9 @@ public void removeExpiredSessionData(Timestamp timestamp) {
                 connection.commit();
             }
         } catch (SQLException e) {
-            log.error("Error while removing Session Data ", e);
+            log.error("Error while removing session data from the database for the timestamp " + timestamp.toString(), e);
         } catch (IdentityException e) {
-            log.error("Error while removing Session Data", e);
+            log.error("Error while obtaining the database connection", e);
```

Comment:
I think logging and trowing an exception inside the getDBConnection is wrong. Logging inside the getDBConnection method has to be removed.  WDYT ?